cos-fleet-catalog-kamelets/mongodb/src/main/resources/kamelets/cos-mongodb-source.kamelet.yaml

```diff
@@ -0,0 +1,115 @@
+    camel.apache.org/kamelet.group: "nosql"
+    camel.apache.org/kamelet.version: "${cos.kamelets.version}"
+  labels:
+    camel.apache.org/kamelet.type: "source"
+spec:
+  definition:
+    title: "MongoDB Source"
+    required:
+      - hosts
+      - collection
+      - database
+    type: object
+    properties:
+      hosts:
+        title: MongoDB Hosts
+        description: A comma-separated list of MongoDB host addresses in `host:port` format.
+        type: string
+      collection:
+        title: MongoDB Collection
+        description: The name of the MongoDB collection to bind to this endpoint.
+        type: string
+      password:
+        title: MongoDB Password
+        description: The user password for accessing MongoDB.
+        type: string
+        format: password
+        x-descriptors:
+          - urn:alm:descriptor:com.tectonic.ui:password
+          - urn:camel:group:credentials
+      username:
+        title: MongoDB Username
+        description: The username for accessing MongoDB. The username must be present in the MongoDB's authentication database (`authenticationDatabase`). By default, the MongoDB `authenticationDatabase` is 'admin'.
+        type: string
+        x-descriptors:
+          - urn:camel:group:credentials
+      ssl:
+        title: Enable ssl for mongodb connection
+        description: whether to enable ssl connection to mongodb
+        type: boolean
+        default: true
+        x-descriptors:
+          - 'urn:alm:descriptor:com.tectonic.ui:checkbox'
+      sslValidationEnabled:
+        title: Enables ssl certificates validation and host name checks.
+        description: IMPORTANT this should be disabled only in test environment since can pose security issues.
+        type: boolean
+        default: true
+        x-descriptors:
+          - 'urn:alm:descriptor:com.tectonic.ui:checkbox'
+      database:
+        title: MongoDB Database
+        description: The name of the MongoDB database.
+        type: string
+      persistentTailTracking:
+        title: MongoDB Persistent Tail Tracking
+        description: Specifies to enable persistent tail tracking, which is a mechanism to keep track of the last consumed data across system restarts. The next time the system is up, the endpoint recovers the cursor from the point where it last stopped consuimg data. This option will only work on capped collections.
+        type: boolean
+        x-descriptors:
+          - 'urn:alm:descriptor:com.tectonic.ui:checkbox'
+        default: false
+      tailTrackIncreasingField:
+        title: MongoDB Tail Track Increasing Field
+        description: The correlation field in the incoming data which is of increasing nature and is used to position the tailing cursor every time it is generated.
+        type: string
+  dependencies:
+    - "camel:core"
+    - "camel:kamelet"
+    - "camel:mongodb"
+    - "camel:jackson"
+  template:
+    beans:
+      - name: local-mongodb
```

Comment:
looking at the kamelet, it looks like this bean is just the vanilla mongodb component hence it can be removed

Reply:
Good catch, I removed the same bean from the mongodb sink kamelet as well